src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +18,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        String exceptionMessage = "Can't get min value from list: ";
+        return numbers.stream()
+                .flatMap(s -> Arrays.stream(s.split(",")))
+                .map(Integer::parseInt)
+                .filter(x -> x % 2 == 0)
+                .min(Integer::compareTo)
+                .orElseThrow(() -> new RuntimeException(exceptionMessage + numbers));
```

Comment:
```suggestion
        return numbers.stream()
                .flatMap(s -> Arrays.stream(s.split(",")))
                .map(Integer::parseInt)
                .filter(x -> x % 2 == 0)
                .min(Integer::compareTo)
                .orElseThrow(() -> new RuntimeException("Can't get min value from list: " + numbers));
```
storing messages in variables is not a good idea - it is hard to read, and not informative

Reply:
agree. but it fails checkstyle compliance because the "line is longer than 100 characters" :man_shrugging: